source/macros.tex

```diff
@@ -242,7 +242,7 @@
 \newcommand{\GrammarStylex}[1]{\textcolor{grammar-gray}{\textsf{\textit{#1}}}}
 
 % Code and definitions embedded in text.
-\newcommand{\tcode}[1]{\CodeStylex{#1}}
+\newcommand{\tcode}[1]{\ifmmode\CodeStylex{#1}\else\CodeStylex{#1}\@\fi}
```

Comment:
Can you add a comment like "restore normal sentence spacing when not in math mode"?